templates/triton_services/README.md

```diff
@@ -0,0 +1,175 @@
+
+The tutorial starts by building a Docker image that can run properly in Anyscale
+platform. You don't need to build the image from scratch. The image used to run this
+tutorial is already used in the workspace. This section is just for informational
+purposes. Some of the code used here are from Nvidia's tutorials from
+[TenserRT Stable Diffusion](https://github.com/NVIDIA/TensorRT/blob/release/10.0/demo/Diffusion/README.md),
+[Triton building complex pipeline](https://github.com/triton-inference-server/tutorials/blob/r24.04/Conceptual_Guide/Part_6-building_complex_pipelines/README.md),
+and [Triton Ray Serve Deployment](https://github.com/triton-inference-server/tutorials/tree/r24.04/Triton_Inference_Server_Python_API/examples/rayserve).
+If you need to learn more about TensorRT and Triton, refer to their official
+documentation and tutorial.
+
+In Nvidia's tutorial, they have an image to build the model and another image to serve
+the model. In this tutorial, the docker image pulls in all the necessary dependencies.
+This image compiles the model, allow you to do local development in Anyscale Workspace,
+and deploys to Anyscale Services.
+
+The full `Dockerfile` used to build the image is already provided in the workspace.
+You can view the file by clicking on the `Dockerfile` in the file explorer. It shows
+what's required in the image to run this tutorial.
+
+## Compile model using Triton's Python backend on Anyscale Workspaces
+
+Unlike Nvidia's tutorial, this tutorial doesn't include models in the Docker image.
+Instead, in this section, you are building the model and upload the model to a cloud
+storage such as AWS S3 or GCP Storage for serving later.
+
+Run this code to start a Triton server using the `/tmp/workspace/diffusion-models`
+directory as the model repository.
+
+
+```python
+import tritonserver
+
+model_repository = ["/tmp/workspace/diffusion-models"]
+
+triton_server = tritonserver.Server(
+    model_repository=model_repository,
+    model_control_mode=tritonserver.ModelControlMode.EXPLICIT,
+)
+triton_server.start(wait_until_ready=True)
+```
+
+Compile the model using Triton's Python backend. This takes 10-15 minutes on a
+T4 GPU and 8-10 minutes on an A10G GPU. The model saves in the `model_repository`
+directory as TensorRT engine artifacts. Also keep in mind the model compiling
+needs to in the same type of GPU you are planning to serve the model on.
+
+
+
+```python
+import time
+import datetime
+
+
+print(f"start time: {datetime.datetime.now()}")
+t0 = time.time()
+model = triton_server.load("stable_diffusion_1_5")
+duration = time.time() - t0
+print(f"Total duration: {duration}s")
+
+# Unload the model and the server to free the memory.
+triton_server.unload(model, wait_until_unloaded=True)
+triton_server.stop()
+```
+
+After the model compile, you can upload the model to a cloud storage. You need to
+upload both the model config file `config.pbtxt` and the TensorRT engine model
```

Comment:
where is the engine model directory? is it the `model_repository` you set earlier?

use consistent language and also would add an explicit comment in the code about where the model will be saved (it's really non-obvious right now)